x/programs/examples/token.go

```diff
@@ -40,8 +40,6 @@ func (t *Token) Run(ctx context.Context) error {
 		"mint_to",
 		"get_balance",
 		"transfer",
-		"alloc",
-		"dealloc",
```

Comment:
I think we can assume these will always be supported so the user should not need to define them. please conform @samliok

Reply:
yea 👍